lib/guardian_db.ex

```diff
@@ -120,4 +135,17 @@ defmodule GuardianDb do
   end
 
   def repo, do: @repo
+
+  defp token_types do
+    Application.get_env(:guardian_db, GuardianDb, [])
```

Comment:
I'm not sure we want to call `Application.get_env(:guardian_db, GuardianDb, [])` and process the values each time it's called.

How about we store the configuration value in a module attribute?  I think it's fair to require the configuration be provided as atoms so we can avoid all the `to_string/1` calls.

```elixir
@token_types = Application.get_env(, GuardianDb, [])

...

defp storable_type?(type, types \\ @token_types) do
defp storable_type?(_, []), do: true
defp storable_type?(type, types), do: Enum.member?(types, type)
```

On a separate note, pipes should start with a bare value:

```elixir
:guardian_db
|> Application.get_env(GuardianDb, [])
...
```